### DISCUSSION TITLE
Floating controls that draw on top of other apps

### DISCUSSION THREAD

**Discussion by kurtnapolitano** [dotnet/maui#9722] [Q&A]
I'm relatively new to MAUI and Mobile development in general.  My google-fu is failing me.  Can someone point me in the direction of how I can build a simple app with non-full-screen "floating" controls that draw on top of other apps, Facebook Messenger's "Chat Heads" feature is a good example of what I'm looking for?

**Comment by xtuzy** [comment]
like this  [How to Make a Floating Window Application in Android ](https://www.youtube.com/watch?v=ATUmOAN_q6A) ? You can google "Floating Window" to find how to do in java, and convert java to C#
